src/FlightBookingApp.Application/Features/Flights/Admin/Command/Update/UpdateCommandValidator.cs

```diff
@@ -0,0 +1,13 @@
+﻿using FlightBookingApp.Application.Features.Flights.Admin.Command.BaseValidator;
+using FluentValidation;
+
+namespace FlightBookingApp.Application.Features.Flights.Admin.Command.Update
+{
+    public class UpdateCommandValidator : AbstractValidator<UpdateFlightCommand>
```

Comment:
I think it's better to include name of the entity into validator's class name to avoid misusage